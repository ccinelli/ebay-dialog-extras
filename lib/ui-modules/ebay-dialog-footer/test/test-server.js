const testUtils = require('../../../helpers/server/test-utils');
const component = require('../');

testUtils.enableSnapshots();

// tests mirrored from eBayUI 

describe('ebay-dialog-footer', () => {
    it('renders basic version', done => {
        testUtils.testSnapshot(component, {
        }, done);
    });
});
