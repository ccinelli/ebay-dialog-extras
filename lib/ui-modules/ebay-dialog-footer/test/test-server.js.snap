// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ebay-dialog-footer renders basic version 1`] = `
"<!--M#s0-->
<div class=\\"flexible-dialog ebay-dialog-footer\\" id=\\"s0\\">
  <div class=\\"ebay-dialog-footer--wrapper\\">
    <div class=\\"ebay-ebay-dialog-flexible--actions\\"></div>
  </div>
</div>
<!--M/-->"
`;
